### PR TITLE
Add type of DREF in operational update

### PR DESCRIPTION
**Pull request by barshathakuri**

### CODE REVIEW

File: src/root/views/DrefOperationalUpdateForm/Operation/index.tsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ function Operation(props: Props) {
   } = props;
 
   const error = getErrorObject(formError);
-  const isImminentOnSet = value.type_of_onset === ONSET_IMMINENT;
+  const isImminentType = value.type_of_onset === TYPE_IMMINENT;
```
Comment (puranban) [inline src/root/views/DrefOperationalUpdateForm/Operation/index.tsx:73]: pass as a props from the parent

File: src/root/views/DrefOperationalUpdateForm/Overview/index.tsx [hunk 2]
```diff
@@ -65,6 +63,9 @@ interface Props {
   onValueSet: (value: SetBaseValueArg<Value>) => void;
   userOptions: NumericValueOption[];
   onCreateAndShareButtonClick: () => void;
+  drefTypeOptions: NumericValueOption[] | undefined;
```
Comment (puranban) [inline src/root/views/DrefOperationalUpdateForm/Overview/index.tsx:66]: remove undefined

File: src/root/views/DrefOperationalUpdateForm/common.tsx [hunk 3]
```diff
@@ -3,9 +3,11 @@ import {
   DistrictMini,
 } from "#types/country";
 
-export const ONSET_IMMINENT = 0;
+export const TYPE_IMMINENT = 0;
 export const ONSET_SLOW = 1;
 export const ONSET_SUDDEN = 2;
+export const TYPE_ASSESSMENT = 3;
+export const TYPE_RESPONSE = 4;
```
Comment (puranban) [inline src/root/views/DrefOperationalUpdateForm/common.tsx:10]: export const ONSET_SLOW = 1;
export const ONSET_SUDDEN = 2;

export const TYPE_IMMINENT = 0;
export const TYPE_ASSESSMENT = 1;
export const TYPE_RESPONSE = 2;

for this take a look at `/dref-options` api

File: src/root/views/DrefOperationalUpdateForm/index.tsx [hunk 4]
```diff
@@ -140,7 +141,7 @@ function DrefOperationalUpdate(props: Props) {
         });
       }
       if (response.images_file?.length > 0) {
-       response.images_file.forEach((img) => {
+        response.images_file.forEach((img) => {
```
Comment (puranban) [inline src/root/views/DrefOperationalUpdateForm/index.tsx:144]: use 2 space for indentation

File: src/root/views/DrefOperationalUpdateForm/index.tsx [hunk 5]
```diff
@@ -317,12 +319,12 @@ function DrefOperationalUpdate(props: Props) {
...
-        needs: false,
-        operation: false,
-        submission: false,
-      };
+      operationOverview: false,
+      eventDetails: false,
+      needs: false,
+      operation: false,
+      submission: false,
+    };
```
Comment (puranban) [inline src/root/views/DrefOperationalUpdateForm/index.tsx:327]: use 2 spacing

**Review by puranban — COMMENTED** [review]
Use 2 tab spaces for the indentation

**Review by frozenhelium — APPROVED** [review]

**Pull request merged**